frontend/public/locales/en/common.json

```diff
@@ -246,6 +246,8 @@
   "more-type": "More {{type}}",
   "verified": "Verified",
   "app-is-verified": "This app is verified",
+  "verification-instructions": "Verify your app ID to indicate that your Flathub package is approved by the upstream developer. You'll need to confirm your identity, for example with a website or an account on a source code hosting site. The verified badge will be shown alongside the app name, publisher name, and the identity you use for verification.",
```

Comment:
Verify your app ID to indicate that your Flathub upload is approved by the app developer. You'll need to prove that you have control of the app via a website on the app's domain or by checking access to the app's account on a source code hosting site. The verified badge will be shown alongside the app name, publisher name, and the way that the app ID was verified.